installer/win/WixSharpInstaller/CloseDlg.cs

```diff
@@ -0,0 +1,57 @@
+using WixSharp;
+using System.Drawing;
+
+namespace Symphony
+{
+    public partial class CloseDlg : WixSharp.UI.Forms.ManagedForm, IManagedDialog
+    {
+
+        public CloseDlg()
+        {
+            InitializeComponent();
+        }
+
+        void dialog_Load(object sender, System.EventArgs e)
```

Comment:
is this an overridden method? We are following both the snake & camel case here. Can we keep it consistent and follow the proper casing as that is the standard for C# ?

Reply:
It is overridden, we can not change the name